app/src/main/java/com/aahlad/thismessagingservice/Model/User.java

```diff
@@ -0,0 +1,45 @@
+package com.aahlad.thismessagingservice.Model;
+
+public class User {
+
+    private String id;
+    private String username;
+    private String imageURL;
```

Comment:
Add all the other fields that get stored by default in a firebase User (email, display name, etc). https://firebase.google.com/docs/reference/android/com/google/firebase/auth/FirebaseUser.html